templates/qa/qa-scans-cd.yml

```diff
@@ -44,4 +44,21 @@ steps:
     - script: echo deal with qa scan relevant to universal-artifact application type
 
   - ${{ if and(ne(lower(parameters.applicationBlueprint),'universal-artifact'), ne(lower(parameters.applicationBlueprint),'nuget-package'), ne(lower(parameters.applicationBlueprint),'azure-function')) }}:
-    - script: echo UNKNOWN application type
\ No newline at end of file
+    - script: echo UNKNOWN application type
+
+# -----------------------------------------------------------------------------------------------------
+# QA AUTOMATION FOR LOWER AND HIGHER (PROD) ENVIRONMENTS STAGE
+# -----------------------------------------------------------------------------------------------------
+- script: echo 'QA CD Lower and Higher Environment Automation Placeholder ***********************************'
+
+- ${{ if eq( lower(parameters.applicationlueprint), 'azure-function' ) }}:
```

Comment:
Spelling error: applicationlueprint is missing the "B"

Reply:
Thank you @AndreasNVI. Fixed.